### ISSUE TITLE
Tkinter links with wrong Tk in Python

### ISSUE THREAD

**Issue by boegel** [hpcugent/easybuild-easyblocks#728]
```
    fig1 = plt.figure(figsize=(32, 18), dpi=60)
  File "/user/scratchdelcatty/gent/gvo000/gvo00002/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/matplotlib/1.4.3-intel-2015b-Python-2.7.10/lib/python2.7/site-packages/matplotlib-1.4.3-py2.7-linux-x86_64.egg/matplotlib/pyplot.py", line 435, in figure
    **kwargs)
  File "/user/scratchdelcatty/gent/gvo000/gvo00002/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/matplotlib/1.4.3-intel-2015b-Python-2.7.10/lib/python2.7/site-packages/matplotlib-1.4.3-py2.7-linux-x86_64.egg/matplotlib/backends/backend_tkagg.py", line 81, in new_figure_manager
    return new_figure_manager_given_figure(num, figure)
  File "/user/scratchdelcatty/gent/gvo000/gvo00002/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/matplotlib/1.4.3-intel-2015b-Python-2.7.10/lib/python2.7/site-packages/matplotlib-1.4.3-py2.7-linux-x86_64.egg/matplotlib/backends/backend_tkagg.py", line 89, in new_figure_manager_given_figure
    window = Tk.Tk()
  File "/user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/Python/2.7.10-intel-2015b/lib/python2.7/lib-tk/Tkinter.py", line 1816, in __init__
    self._loadtk()
  File "/user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/Python/2.7.10-intel-2015b/lib/python2.7/lib-tk/Tkinter.py", line 1832, in _loadtk
    % (_tkinter.TK_VERSION, tk_version)
RuntimeError: tk.h version (8.6) doesn't match libtk.a version (8.5)
```

from the Python install log:

```
-L/usr/local/lib -L. -ltk8.5 -ltcl8.5 -lX11 -lpython2.7 -o build/lib.linux-x86_64-2.7/_tkinter.so
```

while TK 8.6.4 is included as a dependency.

**Comment by boegel** [comment]
See also https://github.com/BrewTestBot/homebrew/commit/5d73eb6481572ff4d6fa9a2f591a08f5fc5f7216

**Comment by stdweird** [comment]
@boegel thx!

**Comment by boegel** [comment]
Fix in #729, let's follow up there.

**Comment by boegel** [comment]
The patch for the Python easyblock to use the `--with-tcltk*` configure fixes this issue, but then another pops up:

```
    window = Tk.Tk()
  File "/user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/CO7/haswell-ib/software/Python/2.7.10-intel-2015b/lib/python2.7/lib-tk/Tkinter.py", line 1814, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: Can't find a usable tk.tcl in the following directories: 
    /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/CO7/haswell-ib/software/Tcl/8.6.4-intel-2015b/lib/tcl8.6/tk8.6 /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/CO7/haswell-ib/software/Tcl/8.6.4-intel-2015b/lib/tk8.6 /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/CO7/haswell-ib/software/Python/2.7.10-intel-2015b/lib/tk8.6 /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/CO7/haswell-ib/software/Python/lib/tk8.6 /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/CO7/haswell-ib/software/Python/2.7.10-intel-2015b/library
```

https://github.com/NixOS/nixpkgs/issues/1479 and https://github.com/NixOS/nixpkgs/commit/decd2feb0a1bc80940e697fa66e3b25383360c30 suggest this is an issue with Tk, rather than Python, will check whether the proposed fix works.

**Comment by boegel** [comment]
Fix for Tk issue is in #2102.
